tests/migrations/test_loader.py

```diff
@@ -48,6 +54,28 @@ def test_apply(self):
             set(),
         )
 
+    def test_has_table_cached(self):
+        """
+        Test that the has_table() method caches a positive result and does not
```

Comment:
To add more explanation, docstrings should state the expected behavior and omit prefixes like "Test that" since all tests test things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).